src/httprpc.cpp

```diff
@@ -66,7 +66,7 @@ static std::string strRPCUserColonPass;
 /* Stored RPC timer interface (for unregistration) */
 static HTTPRPCTimerInterface* httpRPCTimerInterface = 0;
 
-static void JSONErrorReply(HTTPRequest* req, const UniValue& objError, const UniValue& id)
+void JSONErrorReply(HTTPRequest* req, const UniValue& objError, const UniValue& id)
```

Comment:
Why make this non-static? You're not ever using this outside httprpc (same for RPCAuthorized and HTTPReq_HandleAuth)